artifacts/deploy/edgio.yml

```
@@ -35,25 +33,48 @@ jobs:
           fi
       - name: Install dependencies
         if: env.SUCCESS == 'false'
-        run: npm i -g @edgio/cli@latest
+        run: |
+          # Check if package-lock.json file exists
+          if [ -f "package-lock.json" ]; then
+            # Use npm
+            npm i -g @edgio/cli@latest
+          else
+            # Check if yarn.lock file exists
+            if [ -f "yarn.lock" ]; then
+              # Use yarn
+              yarn global add @edgio/cli@latest
+            else
+              # No lock file found
+              echo "No lock file found. Please run 'npm ci' or 'yarn install --frozen-lockfile' to install dependencies."
```

```suggestion
              echo "No lock file found. Please run 'npm install' or 'yarn install' to install dependencies."
```

Updating this to a regular install since `ci` or `--frozen-lockfile` only work if a lockfile exists.

---

Thanks for taking care of the ci part. I forgot to update that one.